docs/source/nas/evaluator.rst

```diff
@@ -42,6 +42,39 @@ The simplest way to customize a new evaluator is with :class:`FunctionalEvaluato
 
     If the conversion is successful, the model will be able to be visualized with powerful tools `Netron <https://netron.app/>`__.
 
+Use Evaluators to Train and Evaluate Models
+-------------------------------------------
+
+Users can use evaluators to train or evaluate a single, concrete architecture. This is very useful when:
+
+* Debugging your evaluator against a baseline model.
+* Fully train, validate and test your model after the search process is complete.
+
+The usage is shown below:
+
+.. code-block:: python
+
+   class Model(nn.Module):
+       ...
+
+   # Use a callable returning a model
+   evaluator.evaluate(Model)
+   # Or initialize the model beforehand
+   evaluator.evaluate(Model())
+
+The underlying implementation of :meth:`~nni.retiarii.Evaluator.evaluate` depends on concrete evaluator that you used.
+For example, if :class:`~nni.retiarii.evaluator.FunctionalEvaluator` is used, it will run your customized fit function.
+If lightning evaluators like :class:`nni.retiarii.evaluator.pytorch.Classification` are used, it will invoke the ``trainer.fit()`` of Lightning.
+
+To evaluate an architecture that is exported from experiment (i.e., from :meth:`~nni.retiarii.experiment.pytorch.RetiariiExperiment.export_top_models`), use :func:`nni.retiarii.fixed_arch` to instantiate the exported model::
+
+    with fixed_arch(exported_model):
+        model = Model()
+    # Then use evaluator.evaluate
+    evaluator.evaluate(model)
```

Comment:
do we need to add the `note` here to tell users that the same `evaluator` instance should not be used both for training supernet and for evaluating a subnet?

Reply:
I think the problem is restricted to pytorch-lightning.

Reply:
No. There is no such implication.